Source/WebSocket.swift

```diff
@@ -188,17 +188,16 @@ public class WebSocket: NSObject, NSStreamDelegate {
      If you supply a zero (or negative) `forceTimeout`, I immediately close the socket (without sending a Close control frame) and notify my delegate.
 
      - Parameter forceTimeout: Maximum time to wait for the server to close the socket.
-     */
-    public func disconnect(forceTimeout forceTimeout: NSTimeInterval? = nil) {
+    */
+    public func disconnect(forceTimeout: TimeInterval? = nil) {
         switch forceTimeout {
-        case .Some(let seconds) where seconds > 0:
-            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(seconds * Double(NSEC_PER_SEC))), callbackQueue) { [weak self] in
+        case .some(let seconds) where seconds > 0:
+            callbackQueue.asyncAfter(deadline: DispatchTime.now() + Double(Int64(seconds * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC)) { [weak self] in
```

Comment:
`callbackQueue.asyncAfter(deadline: DispatchTime.now() + .seconds(seconds)) { [weak self] in`